makefiles/ibc.mk

```diff
@@ -0,0 +1,214 @@
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod agoric-validator agd query ibc client states \
+	'
+
+.PHONY: ibc_list_agoric_connections
+ibc_list_agoric_connections:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod agoric-validator agd query ibc connection connections \
+	'
+
+.PHONY: ibc_list_agoric_channels
+ibc_list_agoric_channels:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod agoric-validator agd query ibc channel channels \
+	'
+
+##########################
+# Remote Balance Queries #
+##########################
+.PHONY: ibc_query_agoric_balance
+ibc_query_agoric_balance:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod agoric-validator agd query bank balances $(AGORIC_ACCOUNT) \
+	'
+
+.PHONY: ibc_query_axelar_balance
+ibc_query_axelar_balance:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod axelar-validator axelard query bank balances $(AXELAR_ACCOUNT) \
+	'
+
+.PHONY: ibc_query_osmosis_balance
+ibc_query_osmosis_balance:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod osmosis-validator osmosisd query bank balances $(OSMOSIS_ACCOUNT) \
+	'
+
+########################
+# IBC transfer targets #
+########################
+
+## Axelar ##
+############
+.PHONY: ibc_test_transfer_axelar_to_pocket
+ibc_test_transfer_axelar_to_pocket:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod axelar-validator \
+			axelard tx ibc-transfer transfer transfer \
+			$${AXELAR_POCKET_SRC_CHANNEL_ID} $(POCKET_ACCOUNT) 1000uaxl \
+			--keyring-backend=test \
+			--chain-id=$(AXELAR_CHAIN_ID) \
+			--from=validator --yes \
+	'
+
+.PHONY: ibc_test_transfer_pocket_to_axelar
+ibc_test_transfer_pocket_to_axelar:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		pocketd --home=$(POCKETD_HOME) tx ibc-transfer transfer transfer \
+		$${POCKET_AXELAR_SRC_CHANNEL_ID} $(AXELAR_ACCOUNT) 1000upokt \
+			--network $(NETWORK) \
+			--keyring-backend=test \
+			--from=app1 --yes \
+	'
+
+## Agoric ##
+############
+.PHONY: ibc_test_transfer_agoric_to_pocket
+ibc_test_transfer_agoric_to_pocket:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		kubectl_exec_grep_pod agoric-validator \
+			agd tx ibc-transfer transfer transfer \
+			$${AGORIC_POCKET_SRC_CHANNEL_ID} $(POCKET_ACCOUNT) 1000ubld \
+			--keyring-backend=test \
+			--chain-id=$(AGORIC_CHAIN_ID) \
+			--from=validator --yes \
+	'
+
+.PHONY: ibc_test_transfer_pocket_to_agoric
+ibc_test_transfer_pocket_to_agoric:
+	bash -c '\
+		source ./tools/scripts/ibc-channels.sh && \
+		pocketd --home=$(POCKETD_HOME) tx ibc-transfer transfer transfer \
+		$${POCKET_AGORIC_SRC_CHANNEL_ID} $(AGORIC_ACCOUNT) 1000upokt \
+			--network=$(NETWORK) \
+			--keyring-backend=test \
+			--from=app1 --yes \
+	'
+
+#############################
```

Comment:
Can you put together a new, small, copy-pasta heavy readme that shows you how develop/debug this on LocalNet with the makefile targets you use?